tests/unit/components/playback/controls/InputWidget.spec.js

```diff
@@ -225,4 +225,27 @@ describe("InputWidget.vue", () => {
       "width: 390px; top: 48px;"
     );
   });
+  test("When the component is mounted, Then then the props value 'display_text_message' is set to 'false' the invalid_text is not rendered, on resetting 'display_text_message' to 'true' then invalid_text is rendered", async () => {
```

Comment:
an initial props configuration should not be in the `Then` section.  The configuration of the props are part of wherever you think the `mounted` should be (either in `Given` or `When`, depending on the test).

So something like `When the component is mounted with the display_text_message prop set to false, Then the invalid_text is not rendered`

Also, you appear to be testing two separate things here (your "Then" clause has become quite long).

I would recommend creating a separate second test: `Given the component was mounted with the display_text_message prop set to false, When the display_text_message prop is updated to True, Then the invalid_text is rendered`

Reply:
Completed.